src/System.Security.Cryptography.Encoding/tests/Asn1/Serializer/SimpleDeserialize.cs

```diff
@@ -724,39 +707,29 @@ public byte[] LittleUInts
 
     [Choice]
     [StructLayout(LayoutKind.Sequential)]
-    public struct DirectoryString
-    {
-        [UTF8String]
-        public string Utf8String;
-        [BMPString]
-        public string BmpString;
-    }
-
-    [Choice]
-    [StructLayout(LayoutKind.Sequential)]
-    public struct FlexibleString
+    struct FlexibleString
     {
-        public DirectoryString? DirectoryString;
+        public DirectoryStringAsn? DirectoryString;
 
         [IA5String]
         public string Ascii;
     }
 
     [Choice(AllowNull = true)]
     [StructLayout(LayoutKind.Sequential)]
-    public sealed class DirectoryStringClass
+    sealed class DirectoryStringClass
```

Comment:
Explicit access modifiers are always required.

Reply:
Ok, will add them. I had to make it internal to align with the other *Asn classes that are now referenced as members.